dhdt/input/read_sentinel2.py

```diff
@@ -906,6 +905,8 @@ def read_detector_mask(path_meta, boi, geoTransform):
             im_meta = f_meta[:-3]+'jp2'
             assert os.path.exists(im_meta), ('gml and jp2 file not present')
             msk = read_geo_image(im_meta)[0]
+            if det_stack is None:
+                det_stack = np.zeros((*msk.shape, len(boi)), dtype='int8')
```

Comment:
isn't ask.shape[::2] better?

Reply:
Right, I  thought  that `read_geo_image` would only read 2D arrays. Will fix this as `mask.shape[:2]`